### ISSUE TITLE
Ubuntu-18.04 Actions runner image is being deprecated

### ISSUE THREAD

**Issue by zann1x** [SOCI/soci#1005]
Just to raise awareness, the runner image we're using will soon be deprecated:
https://github.com/actions/runner-images/issues/6002

**Comment by vadz** [comment]
Yes, we need to either switch to 20.04 or run the builds in 18.04 container if we still want to keep compatibility with it (and we probably do?).

**Comment by vadz** [comment]
Mostly done, but one build still uses 18.04, see #1008.
